Adiciona o post mulheres que impactaram a PyBr2020 (#426)
pyladies-brazil/br-pyladies-pelican: content/2020-11-15-xray-mulheres-que-impactaram-a-python-brasil-2020.md

Title: X-RAY: Mulheres que impactaram a Python Brasil 2020
Slug: xray-mulheres-que-impactaram-a-python-brasil-2020
Date: 2020-11-15 03:40:57
Tags: eventos, python brasil 2020
Author: Lis R. Barreto

<img src="/images/xray-mulheres-pybr/cover-xray-mulheres-pybr2020.png">

A Python Brasil 2020 acabou de acabar e deixou em todas nós um bocadinho de saudade. Pra matar um pouco dessa saudade, resolvi entrevistar algumas das palestrantes que impactaram a maior conferência sobre linguagem de programação Python do Brasil e da América Latina.

A seção X-RAY foi pensada especialmente pra esse post com o objetivo de te deixar por dentro do backstage feminino da PyBr2020 e de enaltecer essas mulheres incríveis. Perdeu a palestra de alguma delas ou quer assistir novamente? Não tem problema. A Python Brasil disponibilizou a gravação de todas as palestras em seu canal do YouTube e você pode acompanhar cada uma através da [grade de programação](https://2020.pythonbrasil.org.br/grade/).

<img src="/images/xray-mulheres-pybr/beyonce-car.gif">

## X-RAY: [Ingrid](https://twitter.com/ingridmurielem) 👑

> A Ingrid participou da PyBr na trilha PEP20 com a palestra **"Como sobreviver a um time que fala um idioma que você não é fluente"**, nessa palestra ela nos trouxe dicas valiosas de sobre carreira e dividiu conosco as suas experiências sobre viajar para um país que não fala o mesmo idioma que você e também como é trabalhar em equipes que se comunicam em um idioma em que você não é fluente.

Reviewer: tentei tirar um pouco a repetição da última frase, mas não sei se ficou muito bom. enfim, só acho que está um pouco repetitivo hahah
```suggestion
A Ingrid participou da PyBr na trilha PEP20 com a palestra **"Como sobreviver a um time que fala um idioma que você não é fluente"**, nessa palestra ela nos trouxe dicas valiosas de sobre carreira e dividiu conosco as suas experiências sobre viajar e trabalhar em um país cujas equipes se comunicam em um idioma em que você não é fluente.
```

[Assista a palestra de Ingrid no YouTube](https://www.youtube.com/watch?v=pB5tYMywt44&t=15891s)

<img src="/images/xray-mulheres-pybr/ingrid.jpg" width="400" height="400">

**1. Quem é a Ingrid enquanto pessoa?**

Ah.. A Ingrid quanto pessoa é uma batalhadora e uma ansiosa que tenta fazer tudo ao mesmo tempo, isso faz ela as vezes esquecer do básico: RESPIRAR. Mas a Ingrid é uma mulher preta e feliz.

**2. Quem é a Ingrid no mundo da Tecnologia?**

> A Ingrid tecnologista é desenvolvedora de software, que tenta criar um ambiente tecnologico mais justo e inclusivo. Tem estudado sobre vies tecnologico e tem questionado outros tecnologistas.

Reviewer: ```suggestion
A Ingrid tecnologista é desenvolvedora de software, que tenta criar um ambiente tecnológico mais justo e inclusivo. Tem estudado sobre viés tecnológico e tem questionado outros tecnologistas.
```

**3. O que te motivou a trabalhar na sua área de atuação?**

A possibilidade de poder apoiar outras pessoas com a criação de tecnologia, quando vi que a tecnologia poderia apoiar as pessoas em desastres naturais e etc. Eu me encantei e resolvi entrar de cabeça.

**4. Deixe seu recadinho para mulheres que querem entrar na área de Tecnologia.**

> Deixe seu recadinho para mulheres que querem entrar na área de Tecnologia.

Reviewer: ```suggestion
```
